src/coreclr/jit/llvmlower.cpp

```diff
@@ -892,20 +892,8 @@ void Llvm::lowerVirtualStubCallAfterArgs(
     // been set up and thus needs a larger shadow stack offset. This is done to not create a new safe point across
     // which GC arguments to the main call would be live; the stub itself may call into managed code and trigger a GC.
     unsigned shadowStackOffsetForStub = getCurrentShadowFrameSize() + shadowArgsSize;
-    GenTree* shadowStackForStub = insertShadowStackAddr(callNode, shadowStackOffsetForStub, _shadowStackLclNum);
-    GenTreePutArgType* shadowStackForStubPutArg =
-        _compiler->gtNewPutArgType(shadowStackForStub, CORINFO_TYPE_PTR, NO_CLASS_HANDLE);
-    INDEBUG(shadowStackForStubPutArg->SetArgNum(-1));
-    CurrentRange().InsertBefore(callNode, shadowStackForStubPutArg);
-
-    GenTree* thisForStub = _compiler->gtNewLclvNode(thisArgLclNum, TYP_REF);
-    GenTreePutArgType* thisForStubPutArg = _compiler->gtNewPutArgType(thisForStub, CORINFO_TYPE_CLASS, NO_CLASS_HANDLE);
-    INDEBUG(thisForStubPutArg->SetArgNum(0));
-    CurrentRange().InsertBefore(callNode, thisForStub, thisForStubPutArg);
-
-    GenTreePutArgType* cellForStubPutArg = _compiler->gtNewPutArgType(cellArgNode, CORINFO_TYPE_PTR, NO_CLASS_HANDLE);
-    INDEBUG(cellForStubPutArg->SetArgNum(1));
-    CurrentRange().InsertBefore(callNode, cellForStubPutArg);
+    GenTree* shadowStackForStub       = insertShadowStackAddr(callNode, shadowStackOffsetForStub, _shadowStackLclNum);
+    GenTree* thisForStub              = _compiler->gtNewLclvNode(thisArgLclNum, TYP_REF);
```

Comment:
```suggestion
    GenTree* shadowStackForStub = insertShadowStackAddr(callNode, shadowStackOffsetForStub, _shadowStackLclNum);
    GenTree* thisForStub = _compiler->gtNewLclvNode(thisArgLclNum, TYP_REF);
    CurrentRange().InsertBefore(callNode, thisForStub);
```
(Indentation changes - since we decided not aligning things)